.github/CODEOWNERS

```diff
@@ -1,8 +1,8 @@
 # Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
 # See https://help.github.com/articles/about-code-owners/
 
-/build/ @natemcmaster
-/eng/ @natemcmaster
+/build/ @dougbu
+/eng/ @dougbu
```

Comment:
Suggest adding entries for `/.azure` and `/.config`

Reply:
added